dbfit-java/core/src/main/java/dbfit/util/TypeNormaliserFactory.java

```diff
@@ -1,50 +1,13 @@
 package dbfit.util;
 
-import java.util.HashMap;
-import java.util.Map;
-
-@SuppressWarnings("unchecked")
 public class TypeNormaliserFactory {
-    private static Map<Class, TypeNormaliser> normalisers = new HashMap<Class, TypeNormaliser>();
-
-    private static Class getCandidateIfAncestor(Class target, Class candidate) {
-        return candidate.isAssignableFrom(target) ? candidate : null;
-    }
-
-    private static Class getCandidateIfBetter(Class currentBest, Class nextCandidate) {
-        return currentBest.isAssignableFrom(nextCandidate) ? nextCandidate : currentBest;
-    }
-
-    private static Class findClosestAncestor(Class targetClass) {
-        Class currentBest = null;
+    private static TypeTransformerFactory ttf = new TypeTransformerFactory();  
```

Comment:
> Where could we host an object that would provide the global transformation/mapping mechanism? In the FitNesseTestHost singleton?

It's the opposite direction I was thinking about - to avoid global hosting it in a global singleton.

> Sounds like a different PR would make this easier.

Yes, let's try to keep this PR simpler